conf/machine/raspberrypi4-64.extra.conf

```diff
@@ -1,5 +1,8 @@
 require conf/machine/include/rpi.inc
 
+# adu compatibility id
+ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "1"
```

Comment:
that's wrong; only for kirkstone-next must be set to 1;

Reply:
when we integrate this PR in kirkstone-next, there it will be set to 2

Reply:
devel image are not compatible to devel-next images anyway, so imho setting it now to 1 and later in kirkstone-next to 2 is sufficient.